content/events/libp2p-day.md

```diff
@@ -0,0 +1,21 @@
+---
+eventName: "libp2p Day"
+eventDescription: "libp2p Day is a gathering of developers, researchers, and engaged community members focused on pushing the boundaries of decentralized peer-to-peer networks. This event is more than just another meetup at ETHDenver. This is where we will dig into turning nuts and bolts solutions and cutting edge research into a usable and robust peer-to-peer networking library for our mobile and ad hoc internet of today." 
+
+eventTopic: "libp2p"
+eventType: "Conference" 
+location: "Denver, United States"
+venueName: "The Slate Denver"
+venueAddress: "1250 Welton St, Denver, CO 80204"
+venueMapsLink: "https://maps.app.goo.gl/WtDBcsachBhb6z2s5"
+website: "https://lu.ma/libp2pday-denver"
+startDate: "02/29/2024"
+endDate: "02/29/2024" 
+tag: "PLN Event" 
+dateTBD: false
+preferredContacts: "Slack|#libp2p-community"
```

Comment:
@laeccles We currently only support the following means as preferredContacts - Twitter, LinkedIn, Discord and email. Please share details relating to one of the suggested means.

Reply:
@WinsCodes I have updated this!